cfme/tests/configure/test_zones.py

```diff
@@ -77,7 +77,11 @@ def test_zone_add_dupe(request):
         description=description)
     request.addfinalizer(zone.delete)
 
-    with error.expected('Name has already been taken'):
+    if current_appliance.version >= 5.9:
+        error_flash = "Name is not unique within region 0"
```

Comment:
**REQUIRED** This should include the region number from the appliance object, and not statically set it

Reply:
@psav, Done! Make sense! 👍